src/main/java/ogc/rs/apiserver/router/routerbuilders/StacRouterBuilder.java

```diff
@@ -90,6 +90,24 @@ void addImplSpecificRoutes() {
         .handler(apiServerVerticle::buildResponse)
         .failureHandler(failureHandler);
 
+    routerBuilder
+        .operation(STAC_ITEMS_API)
+        .handler(stacAssetsAuthZHandler)
+        .handler(apiServerVerticle::auditAfterApiEnded)
+        .handler(apiServerVerticle::getStacItems)
+        .handler(apiServerVerticle::putCommonResponseHeaders)
+        .handler(apiServerVerticle::buildResponse)
+        .failureHandler(failureHandler);
+
+    routerBuilder
+        .operation(STAC_ITEM_BY_ID_API)
+        .handler(stacAssetsAuthZHandler)
```

Comment:
Does not need the authZ handler

Reply:
The Items API should be sitting behind the AuthZ, no?

Reply:
Remove authZ and audit